dashboard/dashboard.go

```diff
@@ -306,6 +313,23 @@ func (db *Dashboard) collectData() {
 			prevDiskRead = curDiskRead
 			prevDiskWrite = curDiskWrite
 
+			// extract metrics from downloaded and push to registry
```

Comment:
@karalabe this is just an example on how to push and update the current block number (and other progress indicators) to the metrics registry, which we experimented with while we were hacking this together with @kurkomisi . It would be good to define what we want on the main dashboard in terms of data/charts/etc.

Reply:
wow